xampp/Containerfile

```diff
@@ -0,0 +1,19 @@
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
+# This example uses rhel-coreos image from 4.13.0-rc.2
+FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:d98795f7932441b30bb8bcfbbf05912875383fad1f2b3be08a22ec148d68607e
+
+#Add the installer file to var/, this can be found at: https://sourceforge.net/projects/xampp/files/XAMPP%20Linux/8.2.4/xampp-linux-x64-8.2.4-0-installer.run/download
+ADD xampp-linux-x64-8.2.0-0-installer.run /var/xampp-linux-x64-8.2.0-0-installer.run
```

Comment:
Hmm, I think it was because the sourceforge download caused re-directs and I couldn't get `curl` to work around it(or I didn't find the right incantation).  I think `wget` worked fine though

Reply:
Ahh, I remember trying that, but I didn't set up an explicit download file for it. Thank you, that works!